### ISSUE TITLE
when using auto_insert_landmarks, it appears to be view problems

### ISSUE THREAD

**Issue by EzTan1** [epfml/landmark-attention#12]
as mem_freq = 63, and seq_length 3535 can't divide 63, view[input_ids.shape[0], -1, mem_freq] would raise error?

**Comment by akmohtashami** [comment]
yes when using the auto_insert_landmarks and the triton kernel for training, the sequence length should be divisible by mem_freq.

**Comment by mkrima** [comment]
yes, when using auto_insert_landmarks and triton kernel during training, sequence length should be divisible by mem_freq.
